README.md

```diff
@@ -44,6 +44,12 @@ Defaults to `*:*:*`.
 * `foo:bar:baz`
 * `a:b:c x:y:z`
 
+##### Optional: opts
+
+If an _opts_ object is provided, it will be checked to see if it contains `extendSession` and its value.
+Specifically, if `extendSession` is set to `false`, then the parameter `X-D2L-Session: no-keep-alive` is sent as part of the token fetch request body.
+This prevents the user session from being extended when the token is refetched.
```

Comment:
I'd suggest making some tweaks to the copy above, and then documenting the options individually.

<pre>
##### scope `String` _(`*:*:*:`)_

If _scope_ is provided, then it will be sent as the request scope of the token.
It should be a properly formatted String, with scopes seperated by spaces.

```js
jwt();
jwt('foo:bar:baz');
jwt('a:b:c x:y:z');
```
</pre>

```suggestion
##### Option: extendSession `Boolean` _(true)_

You may optionally specify that you do not want the user's sessions to be extended by the act of fetching the token.
```